workspaces/src/network/account.rs

```diff
@@ -128,6 +130,19 @@ impl Account {
     ) -> Transaction<'a> {
         Transaction::new(worker.client(), self.signer().clone(), contract_id.clone())
     }
+
+    /// Store the credentials of this account locally in the directory provided.
+    pub fn store_credentials(&self, save_dir: impl AsRef<Path>) -> anyhow::Result<()> {
```

Comment:
we may want this to be async to avoid breaking changes in future when the io is done async, but nbd to just have a breaking change later